.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-macos`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Buffers.dll; SideIndicator=<=}, @{InputObject=System.Diagnostics.Debug.dll; SideIndicator=<=}, @{InputObject=System.Resources.ResourceManager.dll; SideIndicator=<=}, @{InputObject=System.Runtime.Extensions.dll; SideIndicator=<=}, @{InputObject=System.Threading.Tasks.dll; SideIndicator=<=}).
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.316031693749085.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1932
1932:             $pathologicalRatio | Should -BeGreaterThan 5
```